Makefile

```
@@ -160,9 +160,15 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+.PHONY: image-build-common
+image-build-common:
+	$(IMAGE_BUILDER) build --no-cache \
+		--build-arg GOLANG_VERSION=$(TOOLBOX_GOLANG_VERSION) \
+		-f Dockerfiles/$(DOCKERFILE)  $(IMAGE_BUILD_FLAGS) -t $(IMG) .
+
 .PHONY: image-build
-image-build: unit-test ## Build image with the manager.
-	$(IMAGE_BUILDER) build --no-cache -f Dockerfiles/Dockerfile  ${IMAGE_BUILD_FLAGS} -t $(IMG) .
+image-build: DOCKERFILE=Dockerfile
+image-build: unit-test image-build-common ## Build image with the manager.
```

```suggestion
image-build: unit-test image-build-common DOCKERFILE=Dockerfile ## Build image with the manager.
```
is it no possible to pass `DOCKERFILE=Dockerfile` in single line ?

---

No https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html